safe/definitions/exposure_classifications.py

```diff
@@ -673,6 +673,101 @@
     ]
 }
 
+badan_geologi_landcover_classes = {
+    'key': 'volcanic_ash_landcover_classes',
+    'name': tr('Volcanic Ash Landcover Classification'),
```

Comment:
Hi @Gustry and @myarjunar , sorry for the late reply,
I believe it is not only for the volcanic ash, and also can be used for the tsunami hazard because Badan Geologi who uses this classification also develops Tsunami hazard modelling.